packages/airnode-protocol/package.json

```diff
@@ -47,7 +47,6 @@
     "typescript": "^5.0.4"
   },
   "dependencies": {
-    "@api3/airnode-utilities": "^0.11.0",
```

Comment:
Why remove utilities dependendency?

Reply:
This PR adds `airnode-protocol` as dependency of `airnode-validator`, which created cyclic dependency graph that resulted in Airnode failing to build:

```
Lerna (powered by Nx)   The task graph has a circular dependency

   @api3/airnode-admin:build --> @api3/airnode-protocol:build --> @api3/airnode-utilities:build --> @api3/airnode-validator:build --> @api3/airnode-protocol:build
```

The cleanest way to solve this was to remove the `airnode-utilities` dependency since that was only imported for logging within a single script. h/t @amarthadan

Reply:
I see. The utilities package probably should not depend on validator - but that should not be part of this issue.